src/Application.cc

```diff
@@ -275,6 +275,14 @@ bool Application::LoadConfig(const std::string &_config)
     this->dataPtr->windowConfig.MergeFromXML(std::string(printer.CStr()));
   }
 
+  // Closing behavior.
+  if (auto dialogOnExitElem = doc.FirstChildElement("dialog_on_exit"))
+  {
+    bool showDialogOnExit{false};
+    dialogOnExitElem->QueryBoolText(&showDialogOnExit);
+    this->dataPtr->mainWin->SetShowDialogOnExit(showDialogOnExit);
+  }
```

Comment:
I think the previous behavior where `<dialog_on_exit>` was in `<window>` was better. You can move this section to the above `if (auto winElem = doc.FirstChildElement("window"))` and then update this `if` to `if (auto dialogOnExitElem = winElem->FirstChildElement("dialog_on_exit"))`

Reply:
Agree, from a user perspective it is better to have it within the `<window>` node.
Done!